pom.xml

```diff
@@ -33,7 +33,7 @@ THE SOFTWARE.
     </parent>
     <groupId>org.jenkins-ci.plugins.workflow</groupId>
     <artifactId>workflow-multibranch</artifactId>
-    <version>${changelist}</version>
+    <version>783.${changelist}</version>
```

Comment:
As a matter of procedure this should be directly in the branch (conventionally named `783.x` though it should not matter), and then any further changes you wish to make to that branch done in a PR targeting it. https://gist.github.com/jglick/86a30894446ed38f918050c1180483e2 basically. Just to reduce confusion, so that you are mixing up branch _prep_ (a mechanical version number modification) with actual changes to the content of the plugin relative to the 783 release.

Reply:
👍 I can split that out not a problem.

Reply:
https://github.com/jenkinsci/workflow-multibranch-plugin/tree/783.x